src/transformers/models/clvp/modeling_clvp.py

```diff
@@ -1855,13 +1901,27 @@ def generate(
                 priority: 1) from the `generation_config.json` model file, if it exists; 2) from the model
                 configuration. Please note that unspecified parameters will inherit [`~generation.GenerationConfig`]'s
                 default values, whose documentation should be checked to parameterize generation.
+            pad_to_max_mel_tokens (`int`, *optional*):
+                Pads generated speech_ids to the specified value. This is to implement the same logic from the official
+                repo, link: https://github.com/neonbjb/tortoise-tts/blob/80f89987a5abda5e2b082618cd74f9c7411141dc/tortoise/api.py#L430
+                and to make sure the logits are same.
```

Comment:
If this is only used to get the exact same logits, but is less efficient we should also say that for faster sampling we recommend not to use it.

Reply:
yes, let's add a line - "This does not affect generation quality much so please don't consider using it since it is less efficient."